docs/TuxReact.md

```diff
@@ -0,0 +1,449 @@
+# Tux/React
+
+## Table of Contents
+<ol>
+  <li><a href="#Premise">Premise</a></li>
+  <li><a href="#React-Components-in-a-Nutshell">React Components in a Nushell</a>
+    <ol style="list-style-type:upper-alpha">
+      <li><a href="#Writing-React-and-tuxReact-in-JSX">Writing React and tuxReact in JSX</a></li>
+      <li><a href="#Rendering-in-React">Rendering in React</a></li>
+      <li><a href="#React-Components">React Components</a>
+        <ol style="list-style-type:upper-roman">
+          <li><a href="#React-Components-the-Render-Method">the Render Method</a></li>
+          <li><a href="#React-Components-Accessing-the-DOM-from-React">Accessing the DOM from React</a></li>
+          <li><a href="#React-Components-Props-and-State">Props and State</a>
+            <ol>
+              <li><a href="#React-Components-props-children">props.children</a></li>
+              <li><a href="#React-Components-PropTypes">PropTypes</a></li>
+            </ol>
+          </li>
+          <li><a href="#React-Components-Lifecycle-Methods">Lifecycle Methods</a></li>
+        </ol>
+      </li>
+    </ol>
+  </li>
+  <li><a href="#TuxedoJS-and-React">TuxedoJS and React</a>
+    <ol>
+      <li><a href="#TuxedoJS-OwnerClass">TuxedoJS OwnerClass</a></li>
+      <li><a href="#TuxedoJS-OwneeClass">TuxedoJS OwneeClass</a></li>
+      <li><a href="#TuxedoJS-MutableClass">TuxedoJS MutableClass</a></li>
+    </ol>
+  </li>
+</ol>
+
+## <a id="#Premise"></a>Premise [#](#Premise)
+>`React` is an extremely powerful and complex view library. `TuxedoJS` builds on its functionality in order to make it more semantic and easier to work with.
+
+***
+
+### <a id="React-Components-in-a-Nutshell"></a>React Components in a Nutshell [#](#React-Components-in-a-Nutshell)
+React is a view library designed for extremely high performance rendering. The core concept in `React` is the Virtual DOM. When `React` components update they create a virtual version of the new DOM. Then, `React` diffs the new virtual DOM with the old copy and submits only those changes between the two in order to update the real DOM. `React` is a complicated library and we recommend reviewing the full docs [here](http://facebook.github.io/react/). However, we figured it would be valuable tp provide a quick primer on `React` to give an introductory understanding.
+
+***
+
+#### <a id="Writing-React-and-tuxReact-in-JSX"></a>1) Writing React and tux/React in JSX [#](#Writing-React-and-tuxReact-in-JSX)
+`React` can be written beautiful [JSX](http://facebook.github.io/react/docs/displaying-data.html) which allows you to write React components as though they were HTML tags. We love JSX here at TuxedoJS and we stongly recommend using it. Our components are written using JSX but you do not need to write JSX with Tux. Our `React` examples in our docs will be written in JSX syntax.
```

Comment:
Recommend changing the first sentence to be "`React` can be written in beautiful [JSX]..."